src/App.vue

```diff
@@ -3,7 +3,7 @@
     <UiLoading v-if="app.loading || !app.init" class="overlay big" />
     <div v-else>
       <Topnav />
-      <div class="pb-6 overflow-hidden">
+      <div id="content" class="pb-6 overflow-hidden">
```

Comment:
What if you remove overflow-hidden instead of adding "content" with a hardcoded height?

Reply:
cool that worked.. I thought overflow-hidden is used for something else .. but everything seems to be same with out it.. removed now, Thanks